src/Illuminate/Routing/Router.php

```diff
@@ -398,8 +398,8 @@ public static function mergeGroup($new, $old)
             isset($new['where']) ? $new['where'] : []
         );
 
-        if (isset($old['as']) && isset($new['as'])) {
-            $new['as'] = $old['as'].$new['as'];
+        if (isset($old['as'])) {
+            $new['as'] = $old['as'].(isset($new['as']) ? $new['as'] : '');
```

Comment:
Actually that's just a shorthand if so it is either concatinating with `$new['as']` or an empty string.